bin/lint-readme

```diff
@@ -68,4 +68,11 @@ node_modules/.bin/remark \
 cp README.md README.md.temp
 rewrite <README.md.temp >README.md
 
-node_modules/.bin/eslint -- README.md
+module="$(get module-type)"
+if [[ $module == 'esm' ]] ; then
+  source='module'
+else
+  source='script'
+fi
```

Comment:
All three pairs of single quotes can be dropped. :)

Reply:
:zap: